src/module_3/module_3.md

```diff
@@ -0,0 +1,891 @@
+      <th>0</th>
+      <td>33826472919172</td>
+      <td>ricepastapulses</td>
+      <td>2807985930372</td>
+      <td>3482464092292</td>
+      <td>2020-10-05 16:46:19</td>
+      <td>2020-10-05 00:00:00</td>
+      <td>3</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>...</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>2.0</td>
+      <td>33.0</td>
+      <td>42.0</td>
+      <td>31.134053</td>
+      <td>30.0</td>
+      <td>30.0</td>
+      <td>24.27618</td>
+    </tr>
+    <tr>
+      <th>1</th>
+      <td>33826472919172</td>
+      <td>ricepastapulses</td>
+      <td>2808027644036</td>
+      <td>3466586718340</td>
+      <td>2020-10-05 17:59:51</td>
+      <td>2020-10-05 00:00:00</td>
+      <td>2</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>...</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>2.0</td>
+      <td>33.0</td>
+      <td>42.0</td>
+      <td>31.134053</td>
+      <td>30.0</td>
+      <td>30.0</td>
+      <td>24.27618</td>
+    </tr>
+    <tr>
+      <th>2</th>
+      <td>33826472919172</td>
+      <td>ricepastapulses</td>
+      <td>2808099078276</td>
+      <td>3481384026244</td>
+      <td>2020-10-05 20:08:53</td>
+      <td>2020-10-05 00:00:00</td>
+      <td>4</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>...</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>2.0</td>
+      <td>33.0</td>
+      <td>42.0</td>
+      <td>31.134053</td>
+      <td>30.0</td>
+      <td>30.0</td>
+      <td>24.27618</td>
+    </tr>
+    <tr>
+      <th>3</th>
+      <td>33826472919172</td>
+      <td>ricepastapulses</td>
+      <td>2808393957508</td>
+      <td>3291363377284</td>
+      <td>2020-10-06 08:57:59</td>
+      <td>2020-10-06 00:00:00</td>
+      <td>2</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>...</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>2.0</td>
+      <td>33.0</td>
+      <td>42.0</td>
+      <td>31.134053</td>
+      <td>30.0</td>
+      <td>30.0</td>
+      <td>24.27618</td>
+    </tr>
+    <tr>
+      <th>4</th>
+      <td>33826472919172</td>
+      <td>ricepastapulses</td>
+      <td>2808429314180</td>
+      <td>3537167515780</td>
+      <td>2020-10-06 10:37:05</td>
+      <td>2020-10-06 00:00:00</td>
+      <td>3</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>...</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>2.0</td>
+      <td>33.0</td>
+      <td>42.0</td>
+      <td>31.134053</td>
+      <td>30.0</td>
+      <td>30.0</td>
+      <td>24.27618</td>
+    </tr>
+  </tbody>
+</table>
+<p>5 rows × 27 columns</p>
+</div>
+
+
+
+### Filtering for orders with 5 or more items
+
+
+```python
+items_per_order = df.groupby('order_id')['outcome'].sum()
+big_orders = items_per_order[items_per_order >= 5]
+orders_nums = big_orders.index
```

Comment:
as a rule of thumb, I'd never use `index`. Instead use a unique identifier like `order_id`